webui/react/package.json

```diff
@@ -36,6 +36,7 @@
     "pretty-bytes": "^5.4.1",
     "query-string": "^7.0.0",
     "react": "^17.0.2",
+    "react-contenteditable": "^3.3.5",
```

Comment:
@hamidzr I cannot get `lockfileVersion` to stay 2. That's supported only since npm > 7 and if i use the last versions coming with node 15 or 16 I get either a broken app (`make live`) or a broken deps install (`npm install`).

Lmk if you have suggestions.

Reply:
same here. this looks good thanks for looking into it